src/constraint_system/standard/proof.rs

```diff
@@ -274,12 +273,11 @@ impl Proof {
     // Commitment to [r]_1
     fn compute_linearisation_commitment(
         &self,
-        g: &G1Affine,
         alpha: &Scalar,
         beta: &Scalar,
         gamma: &Scalar,
         z_challenge: &Scalar,
-        l1_eval: &Scalar,
+        l1_eval: Scalar,
```

Comment:
Why is it not taking a `&Scalar` as the other items??

Reply:
Because it is not used after this point, so we can make the function consume the value

Reply:
Perfect!